packages/create-vite/template-vanilla/counter.js

```diff
@@ -0,0 +1,9 @@
+export function setupCounter(element) {
```

Comment:
Maybe it is good to use the same file structure as `template-vanilla-ts` ? (files in `src` directory)

Reply:
I thought the same, but looks like the vanilla template without ts was left flat to further show the most barebone possible structure. I think it isn't bad to leave it as is.

Reply:
That sounds good to leave it. 👍